src/pages/search/index.js

```diff
@@ -400,9 +331,13 @@ export class Search extends Component {
         'creator_id'
       ),
     })
-    //this.latest(999999)
   }
-
+  componentDidMount = async () => {
```

Comment:
why is it async?

Reply:
My bad, I though it was a requirement of CdM